lib/Mount/GroupFolderStorage.php

```diff
@@ -85,7 +87,14 @@ public function getScanner($path = '', $storage = null) {
 		if (!$storage) {
 			$storage = $this;
 		}
-		if (!isset($storage->scanner)) {
+		$isObjectStoreStorage = false;
+		$storageIterator = $storage;
+		while ($storageIterator instanceof Wrapper) {
```

Comment:
you can use $storage->instanceOfStorage() so you don't need to iterate

Reply:
Nice, changed :D